test/jit/test_torchbind.py

```diff
@@ -26,13 +31,8 @@ class TestTorchbind(JitTestCase):
     def setUp(self):
         if IS_SANDCASTLE or IS_WINDOWS or IS_MACOS or IS_FBCODE:
             raise unittest.SkipTest("non-portable load_library call used in test")
-        if TEST_WITH_ROCM:
```

Comment:
I believe it is okie to remote TEST_WITH_ROCM or IN_CI condition, given when shared library file is available in install folder, we could use them first, other wise ones in build folder. The logic works for both internal CI or dev local